### PR TITLE
Add openh264

### PR DESCRIPTION
**Pull request by 183amir**
This could also potentially fix https://github.com/conda-forge/ffmpeg-feedstock/issues/44

### CODE REVIEW

**Comment by 183amir** [comment]
@conda-forge/staged-recipes the linter is failing here. Do you know why?

**Comment by 183amir** [comment]
on the mac the linkage is wrong:
```
+ conda inspect linkages -p /Users/travis/miniconda/conda-bld/openh264_1518377404990/_t_env openh264
openh264
--------
system:
    libSystem.B.dylib (/usr/lib/libSystem.B.dylib)
    libc++.1.dylib (/usr/lib/libc++.1.dylib)
    libopenh264.4.dylib (/usr/local/lib/libopenh264.4.dylib)
```
Does anyone know how can I fix this? (`libopenh264.4.dylib (/usr/local/lib/libopenh264.4.dylib)`)

**Comment by 183amir** [comment]
@isuruf would you happen to have any ideas on the mac linkage?

**Comment by isuruf** [comment]
Can you add `otool -L $PREFIX/lib/libopenh264.1.7.0.dylib` at the end of the build script, so that I can see the output in travis?

File: recipes/openh264/build.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-make -j${CPU_COUNT}
+make PREFIX=$PREFIX -j${CPU_COUNT}
```
Comment (isuruf) [inline recipes/openh264/build.sh]: +1. Looks good. `install-name` should be fixed because of this

**Comment by 183amir** [comment]
Thank you for your help @isuruf I think it is ready for merge now.

**Pull request merged**